templates/cisco_ios_show_interfaces_description.template

```diff
@@ -0,0 +1,11 @@
+Value PORT (\S+)
+Value STATUS (up|down|admin down)
+Value PROTOCOL (up|down)
+Value DESCRIP (.+?)
+
+Start
+  ^Interface -> Begin
+
+Begin
+  ^${PORT}\s+${STATUS}\s+${PROTOCOL}\s+${DESCRIP}$$ -> Record
```

Comment:
```suggestion
  ^${PORT}\s+${STATUS}\s+${PROTOCOL}(?:\s+${DESCRIP})?\s*$$ -> Record
```
Allows for a trailing space
Allows for `DESCRIP` to be empty and not record a space